docs/deployment/high_availability_guide.md

```diff
@@ -29,9 +29,9 @@ Load balancing aims to optimize all Kyuubi service units usage, maximize through
 
 ![](../imgs/ha.png)
 
-With Hive JDBC Driver, a client can specify service discovery mode in JDBC connection string, i.e. `serviceDiscoveryMode=zooKeeper;` and set `zooKeeperNameSpace=kyuubiserver;`, then it can randomly pick one of the Kyuubi service uris from the specified ZooKeeper address in the `/kyuubiserver` path.
+With Hive JDBC Driver, a client can specify service discovery mode in JDBC connection string, i.e. `serviceDiscoveryMode=zooKeeper;` and set `zooKeeperNamespace=kyuubi;`, then it can randomly pick one of the Kyuubi service uris from the specified ZooKeeper address in the `/kyuubi` path.
 
-When we set `kyuubi.ha.enabled` to `true`, load balance mode is activated by default. Please make sure that you specify the correct ZooKeeper address via `kyuubi.ha.zookeeper.quorum` and `kyuubi.ha.zookeeper.client.port`.
+When we specify the ZooKeeper address via `kyuubi.ha.zookeeper.quorum`, load balance mode is activated by default. Please make sure that you specify the correct ZooKeeper address via `kyuubi.ha.zookeeper.quorum` and `kyuubi.ha.zookeeper.client.port`.
```

Comment:
Can we also update L38

Reply:
the configurations listed here are out of date, removed, and add a link to https://kyuubi.apache.org/docs/latest/deployment/settings.html#ha.